lib/ae_mdw_web/controllers/aex9_controller.ex

```diff
@@ -377,17 +384,20 @@ defmodule AeMdwWeb.Aex9Controller do
          account_pk,
          {kbi, mbi} = block_index
        ) do
-    {:ok, Model.block(tx_index: block_txi, hash: block_hash)} =
-      State.get(state, Model.Block, block_index)
+    {:ok, Model.block(hash: block_hash)} = State.get(state, Model.Block, block_index)
```

Comment:
Can we use `State.fetch!/3` here?

Reply:
The `block_index` is validated before but changed.